src/connection-server.ts

```diff
@@ -36,7 +36,7 @@ export function* createConnectionServer(mail: Mailbox, options: ConnectionServer
     yield fork(function* sendRun() {
       yield sendData(connection, JSON.stringify({
         type: "open",
-        url: `http://localhost:${options.proxyPort}`,
+        url: `http://localhost:${options.proxyPort}?${Date.now()}`,
```

Comment:
Is it better to use a UUID here? Maybe there is a way to set the `cache-control` header that works? I'm not super stoked about this, but not sure what the best path is either 🤷‍♂